### PR TITLE
feat(owners): support a body for claim/unclaim

### PR DESCRIPTION
**Pull request by daddykotex**
Fixes MM-6997

### CODE REVIEW

**Comment by daddykotex** [comment]
Very minor

File: smartobjects/ingestion/owners.py [hunk 1]
```diff
@@ -31,33 +31,51 @@ def create(self, owner):
         self._validate_owner(owner)
         self.api_manager.post('owners', owner)
 
-    def claim(self, username, device_id):
+    def claim(self, username, device_id, optionalBody = {}):
```
Comment (mvonthron) [inline smartobjects/ingestion/owners.py]: Use `None` to indicate an optional parameter

Comment (daddykotex) [inline smartobjects/ingestion/owners.py]: Ahhh did not know that, thanks

**Comment by daddykotex** [comment]
I've updated the default.

When I'm done, with all the SDKs (hopefully today). I'll update all of the READMEs.

**Pull request merged**